meta/src/main/proto/gravitino_meta.proto

```diff
@@ -101,3 +101,10 @@ message Topic {
   map<string, string> properties = 4;
   AuditInfo audit_info = 5;
 }
+
+message User {
+  uint64 id = 1;
+  string name = 2;
+  map<string, string> properties = 3;
+  AuditInfo audit_info = 4;
```

Comment:
Do you need to add some more basic information for user?

Reply:
I hesitate to add them. If some properties are very important, we can make them become fields. At the beginning, we can put them into the properties.